### ISSUE TITLE
cat not echo in example

### ISSUE THREAD

**Issue by mblarsen** [marionebl/svg-term-cli#12]
I may be mistaken but shouldn't the examples from `svg-term --help` use `cat` not `echo`:

```
Examples
    $ echo rec.json | svg-term
```

vs

```
Examples
    $ cat rec.json | svg-term
```

**Comment by marionebl** [comment]
True, do you want to help out with a PR?

**Comment by mblarsen** [comment]
Sure

**Comment by marionebl** [comment]
Released with v2.0.4
